set.mm

```diff
@@ -237805,6 +238079,472 @@ factorization into prime power factors (even if the exponents are
+  $}
+
+  ${
+    issimpgd.1 $e |- ( ph -> G e. Grp ) $.
+    issimpgd.2 $e |- ( ph -> ( NrmSGrp ` G ) ~~ 2o ) $.
+    $( Deduce a simple group from its properties.  (Contributed by Rohan
+       Ridenour, 3-Aug-2023.) $)
+    issimpgd $p |- ( ph -> G e. SimpGrp ) $=
+      ( cgrp wcel cnsg cfv c2o cen wbr csimpg issimpg sylanbrc ) ABEFBGHIJKBLFC
+      DBMN $.
+  $}
+
+  $( A simple group is a group.  (Contributed by Rohan Ridenour,
+     3-Aug-2023.) $)
+  simpggrp $p |- ( G e. SimpGrp -> G e. Grp ) $=
+    ( csimpg wcel cgrp cnsg cfv c2o cen wbr issimpg simplbi ) ABCADCAEFGHIAJK
+    $.
+
+  ${
+    simpggrpd.1 $e |- ( ph -> G e. SimpGrp ) $.
+    $( A simple group is a group.  (Contributed by Rohan Ridenour,
+       3-Aug-2023.) $)
+    simpggrpd $p |- ( ph -> G e. Grp ) $=
+      ( csimpg wcel cgrp simpggrp syl ) ABDEBFECBGH $.
+  $}
+
+  $( A simple group has two normal subgroups.  (Contributed by Rohan Ridenour,
+     3-Aug-2023.) $)
+  simpg2nsg $p |- ( G e. SimpGrp -> ( NrmSGrp ` G ) ~~ 2o ) $=
+    ( csimpg wcel cgrp cnsg cfv c2o cen wbr issimpg simprbi ) ABCADCAEFGHIAJK
+    $.
+
+  ${
+    trivnsimpgd.1 $e |- B = ( Base ` G ) $.
+    trivnsimpgd.2 $e |- .0. = ( 0g ` G ) $.
+    trivnsimpgd.3 $e |- ( ph -> G e. Grp ) $.
+    trivnsimpgd.4 $e |- ( ph -> B = { .0. } ) $.
+    $( Trivial groups are not simple.  (Contributed by Rohan Ridenour,
+       3-Aug-2023.) $)
+    trivnsimpgd $p |- ( ph -> -. G e. SimpGrp ) $=
+      ( cnsg cfv c2o cen wbr csimpg wcel csn snnen2o trivnsgd breq1d mtbiri
+      simpg2nsg nsyl ) ACIJZKLMZCNOAUDBPZKLMBQAUCUEKLABCDEFGHRSTCUAUB $.
+  $}
+
+  ${
+    simpgntrivd.1 $e |- B = ( Base ` G ) $.
+    simpgntrivd.2 $e |- .0. = ( 0g ` G ) $.
+    simpgntrivd.3 $e |- ( ph -> G e. SimpGrp ) $.
+    $( Simple groups are nontrivial.  (Contributed by Rohan Ridenour,
+       3-Aug-2023.) $)
+    simpgntrivd $p |- ( ph -> -. B = { .0. } ) $=
+      ( csn wceq csimpg wcel adantr cgrp simpggrpd simpr trivnsimpgd pm2.65da
+      wa ) ABDHIZCJKZATSGLASRBCDEFACMKSACGNLASOPQ $.
+  $}
+
+  ${
+    $d x .0. $.  $d x B $.
+    simpgnideld.1 $e |- B = ( Base ` G ) $.
+    simpgnideld.2 $e |- .0. = ( 0g ` G ) $.
+    simpgnideld.3 $e |- ( ph -> G e. SimpGrp ) $.
+    $( A simple group contains a nonidentity element.  (Contributed by Rohan
+       Ridenour, 3-Aug-2023.) $)
+    simpgnideld $p |- ( ph -> E. x e. B -. x = .0. ) $=
+      ( cv wceq wral wn wrex csn simpgntrivd c0 wne wb cgrp wcel cmnd simpggrpd
+      grpmnd mndidcl 3syl ne0d eqsn syl mtbid rexnal sylibr ) ABIEJZBCKZLULLBCM
+      ACENJZUMACDEFGHOACPQUNUMRACEADSTDUATECTADHUBDUCCDEFGUDUEUFBCEUGUHUIULBCUJ
+      UK $.
+  $}
+
+  ${
+    enpr2d.1 $e |- ( ph -> A e. C ) $.
+    enpr2d.2 $e |- ( ph -> B e. D ) $.
+    enpr2d.3 $e |- ( ph -> -. A = B ) $.
+    $( A pair with distinct elements is equinumerous to ordinal two.
+       (Contributed by Rohan Ridenour, 3-Aug-2023.) $)
+    enpr2d $p |- ( ph -> { A , B } ~~ 2o ) $=
```

Comment:
This theorem should be moved up (near after ~en2sn?)

Reply:
Moved it right after the block where `en2sn` is.